cli/testdata/server-config.yaml.golden

```diff
@@ -0,0 +1,336 @@
+networking:
+    # The URL that users will use to access the Coder deployment.
+    # (default: <unset>, type: url)
+    accessURL:
+    # Specifies the wildcard hostname to use for workspace applications in the form
+    # "*.example.com".
+    # (default: <unset>, type: url)
+    wildcardAccessURL:
+    # Specifies whether to redirect requests that do not match the access URL host.
+    # (default: <unset>, type: bool)
+    redirectToAccessURL: false
+    http:
+        # HTTP bind address of the server. Unset to disable the HTTP endpoint.
+        # (default: 127.0.0.1:3000, type: string)
```

Comment:
small nit: I think standard YAML indentation is 2 spaces. Whenever I use `--write-config` my editor just changes all the indentation when saving.

Reply:
Good catch